prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        return f"{self.name}, fuel={self.fuel}, odometer={self._odometer}"
```

Comment:
It may be beneficial to put "Car:" in the return function to make it easier to identify what __str__ is returning.
eg :  return f"Car: {self.name}, fuel={self.fuel}, odometer={self._odometer}"